public/data/OmAvhad.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Om Avhad",
+  "type": "personal",
+  "bio": "Backend Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/89376666?v=4",
```

Comment:
```suggestion
  "avatar": "https://github.com/OmAvhad.png",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix